pkg/api/operation.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
```

Comment:
Is there a more specific package for these operation structures?

Reply:
This one will be used in observer to unmarshal operation so it is proto-like

Reply:
Moved to api/batch, also moved protocol to api/protocol